controllers/nnf_workflow_controller_helpers.go

```diff
@@ -118,3 +121,49 @@ func addDirectiveIndexLabel(object metav1.Object, index int) {
 	labels[nnfv1alpha1.DirectiveIndexLabel] = strconv.Itoa(index)
 	object.SetLabels(labels)
 }
+
+// Wait on the NnfAccesses for this workflow-index to reach the provided state.
+func (r *NnfWorkflowReconciler) waitForNnfAccessStateAndReady(ctx context.Context, workflow *dwsv1alpha1.Workflow, index int, state string) (*ctrl.Result, error) {
+
+	accessSuffixes := []string{"-computes"}
+
+	fsType, err := r.getDirectiveFileSystemType(ctx, workflow, index)
+	if err != nil {
+		return nil, nnfv1alpha1.NewWorkflowError("Unable to determine directive file system type").WithError(err)
+	}
+
+	if fsType == "gfs2" || fsType == "lustre" {
```

Comment:
I've noticed this sort of thing in a number of places. Are we going to be supporting additional filesystems at any point? If so, we should keep a list of supported fs so we don't need to do these explicit checks all over the place.

Reply:
I've been wanting to change this to a type for the longest time. Then we can do `fsType.IsShared()`

But, that being said - LLNL doesn't require any more file systems from _us_, but they want to use containers to manage _any_ file system i.e. GPFS, UnifyFS